singer-io/tap-dynamodb
# Added None check in deserialize.py

Avoids the scenario where NewImage is not found on the stream record resulting in an attempt to deserialize the items on NoneType ie. {'M': None} now calls as {'Null':True} instead.

# Description of change
See issue https://github.com/singer-io/tap-dynamodb/issues/30 for details on fix

# Manual QA steps
Configure DynamoDB Stream with StreamViewType : "KEYS_ONLY". Configure LOG_BASED Stitch extraction of DynamoDB stream.
When the extraction fails the following error message should be seen: Dynamo stream view type must be either "NEW_IMAGE" "NEW_AND_OLD_IMAGES NOT Extraction failed for ‘NoneType’ object has no attribute ‘items'
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

# Additional Info
This is actually a duplicate PR to https://github.com/singer-io/tap-dynamodb/pull/31 however this PR was closed as I deleted the linked Github account and didn't transfer the repo to my new account before hand.
